src/traces/sankey/render.js

```diff
@@ -10,6 +10,7 @@
 
 var c = require('./constants');
 var d3 = require('d3');
+var sum = require('d3-array').sum;
```

Comment:
Ha this here only works between `d3-sankey` already depends on `d3-array`.

![image](https://user-images.githubusercontent.com/6675409/52823047-9c238a00-3081-11e9-8287-3a8e76ded3fd.png)

We should add `d3-array` to the (plotly.js) dependencies OR use plain loops and save one line in our `package.json`.

Reply:
I think I will do the latter and save one line in our `package.json`.

I should probably replace all the `forEach` while I'm at it right?

Reply:
> I should probably replace all the `forEach` while I'm at it right?

That would be amazing :muscle:

Reply:
Done in c228b37